src/redux/categories/categories.js

```diff
@@ -0,0 +1,20 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] Similarly, assigning value to _CHECK_STATUS_ action type with respect to the _duck patterns approach_ should be something like below:
```javascript
      CHECK_STATUS = bookstore/categories/CHECK_STATUS;
```
 
Kindly ensure to improve upon this aspect.